INSTALL.md

```diff
@@ -0,0 +1,89 @@
+This file gives a detailed explanation of how to install JobAdder on user clients, the server, or worker clients.
+Note that all components of JobAdder were designed to be used exclusively on Linux.
+Furthermore, the server and the worker clients are assumed to be using central authentication (e.g. LDAP).
+
+## Requirements
+The following software is required to run JobAdder:
+
+- Python 3.7 or higher (user client, server, and worker clients)
+- PostgreSQL (server)
+- Docker (worker clients)
+
+## Installation Notes (Linux)
+Detailed step-by-step instructions to install JobAdder.
+
+### 1. Install JobAdder
+The git repository can be cloned by running:
+    
+    git clone https://github.com/DistributedTaskScheduling/JobAdder
+
+After that, execute the *install.sh* script located in the git repository's root directory with superuser privileges.
+This will install JobAdder system-wide through pip.
+
+### 2. Post-Installation Instructions
+Additional steps to take after the installation of JobAdder.
+#### 2.1 Post-Installation Instructions (Server)
+Additional steps to take on the server.
+##### 2.1.1 Create the jobadder User and Group
+Add the *jobadder* user and the corresponding group by running:
+
+    sudo useradd --system jobadder
+
+The jobadder user is used by the daemons running on the server and the worker clients.
+Users in the jobadder group are granted administrative privileges in the context of JobAdder: 
+they can manage worker clients and the jobs of any user.
+
+The jobadder user needs to be added to the docker group to run jobs:
+
+    sudo usermod -a -G docker jobadder
+
+##### 2.1.2 Configure PostgreSQL
```

Comment:
The user may need to adjust PostgreSQL authentication permissions, depending on the distribution's default settings. Something like `https://github.com/DistributedTaskScheduling/documentation/pull/88/files#diff-39e2ddcedb8cf74ada971b3371c4206fR20-R26` (maybe only as a tip, but it would save time for somebody unfamiliar with the database)